apps/smbmount.sh

```diff
@@ -0,0 +1,296 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Check if root
+root_check
+
+# Install cifs-utils
+install_if_not cifs-utils
+
+# Secure fstab
+if [ "$(stat -c %a /etc/fstab)" != "600" ]
+then
+    chmod 600 /etc/fstab
+fi
+# Variables
+SMBSHARES="/mnt/smbshares"
+SMBSHARES_SED="\/mnt\/smbshares"
+
+# Functions
+add_mount() {
+# Check if mounting slots are available
+if grep -q "$SMBSHARES/1" /etc/fstab && grep -q "$SMBSHARES/2" /etc/fstab && grep -q "$SMBSHARES/3" /etc/fstab
+then
+    msg_box "All three slots are occupied. No mounting slots available. Please delete one of the SMB-mounts."
+    return
+fi
+# Enter SMB-server and Share-name
+while true
+do
+    SERVER_SHARE_NAME=$(whiptail --inputbox "Please Enter the server and Share-name like this:\n//Server/Share" "$WT_HEIGHT" "$WT_WIDTH" 3>&1 1>&2 2>&3)
```

Comment:
should enter be with capital "E"?

Reply:
No, but that's actually something I haven't check. I will now read the whole script again for spelling mistakes.

Reply:
This is hopefully done. Please tell me, if you find more.